src/web/app/pages-instructor/instructor-home-page/instructor-home-page.component.html

```diff
@@ -39,14 +39,14 @@ <h1>Home</h1>
   <div *tmIsLoading="!hasCoursesLoaded || isCopyLoading">
     <div *ngIf="courseTabModels.length > 0">
       <div id="course-tab-{{ idx }}" class="card" *ngFor="let courseTabModel of courseTabModels; let idx = index">
-        <div class="card-header bg-primary text-white cursor-pointer"
+        <div class="card-header bg-primary text-white cursor-pointer overflow-hidden"
           (click)="courseTabModel.isTabExpanded = handleClick($event, courseTabModel); this.loadFeedbackSessions(idx);">
           <b class="course-details text-break">[{{ courseTabModel.course.courseId }}]: {{
             courseTabModel.course.courseName }}</b>
           <div class="card-header-btn-toolbar flex-lg-shrink-0" *ngIf="courseTabModel.isAjaxSuccess">
-            <span ngbDropdown>
+            <span ngbDropdown container="body">
               <button type="button" class="btn btn-primary btn-sm" ngbDropdownToggle> Students </button>
-              <div ngbDropdownMenu (click)="$event.stopPropagation()">
+              <div ngbDropdownMenu  (click)="$event.stopPropagation()">
```

Comment:
Let's remove unnecessary formatting changes - not sure why a space here is needed

Reply:
Definitely, I will format all the similar scenarios